packages/amplication-server/src/core/organization/sampleAppData.ts

```diff
@@ -0,0 +1,244 @@
+import { Entity, EntityField } from 'src/models';
```

Comment:
The name sample is misleading. Please name it default, initial, or something else